lib/playground.dart

```diff
@@ -187,7 +191,12 @@ class Playground {
 
     keys.bind('ctrl-s', _handleSave);
     keys.bind('ctrl-enter', _handleRun);
-    keys.bind('f1', _handleHelp);
+    keys.bind('f1', () => _toggleDocTab());
+    document.onKeyUp.listen((e) => _handleHelp());
+    document.onClick.listen((e) => _handleHelp());
+
+    querySelector("#doctab").onClick.listen((e) => _toggleDocTab());
```

Comment:
Not for this PR, but now that we have tab like things in two places, I'll write a Dart wrapper class for a tab component.